packages/atomic/src/components/search-box-suggestions/atomic-search-box-recent-queries/atomic-search-box-recent-queries.tsx

```diff
@@ -0,0 +1,56 @@
+import {buildRecentQueriesList} from '@coveo/headless';
+import {Component, Element, Prop, State, h} from '@stencil/core';
+import {dispatchSearchBoxSuggestionsEvent} from '../suggestions-common';
+
+@Component({
+  tag: 'atomic-search-box-recent-queries',
+  shadow: true,
+})
+export class AtomicSearchBoxRecentQueries {
+  @Element() private host!: HTMLElement;
+
+  @State() public error!: Error;
+
+  @Prop() public maxWithQuery?: number;
```

Comment:
Similar as above, why do we need two prop for this ?

Reply:
https://github.com/coveo/ui-kit/pull/1261/files#r717899897